amethyst_renderer/src/pipe/stage.rs

```diff
@@ -38,16 +38,29 @@ pub struct Stage {
     clear_color: Option<[f32; 4]>,
     clear_depth: Option<f32>,
     enabled: bool,
-    passes: Vec<CompiledPass>,
+    /// Draw passes of the draw stage
+    pub passes: Vec<CompiledPass>,
     target: Target,
+	target_name: String,
 }
 
 impl Stage {
     /// Builds a new `Stage` which outputs to the `Target` with the given name.
     pub fn with_target<N: Into<String>>(target_name: N) -> StageBuilder {
         StageBuilder::new(target_name.into())
     }
-
+	/// Updates the stage target to a copy of some pipe.target
+	pub fn update_target(&mut self,updated:Target){
```

Comment:
Could we just make this a setter or is there anything else that could qualify this as an "update" in the future as opposed to a simple write?

**EDIT:** How does this differ from `set_target`?